### PR TITLE
Compat toolchain upgrade

### PR DESCRIPTION
**Pull request by Ferruck**
This pulls all parts of the compat toolchain up to the versions used in the last supported Ubuntu LTS 18.04.6. The version of the GCC used previously was too old to compile recent U-Boot tools. Additionally, this would probably be a fix for #123 (haven't tested that, @mahaase could you give it a spin?).

### CODE REVIEW

**Comment by Ferruck** [comment]
I'm still testing these changes with our AppImage recipes but I expect no problems. I'll report back.

File: recipes/libs/compat/glibc.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ buildScript: |
             --prefix=/usr                    \
             --host=${AUTOCONF_TARGET}        \
             --build=$($1/scripts/config.guess) \
-            --enable-kernel=4.3.3             \
+            --enable-kernel=5.4.157          \
```
Comment (jkloetzke) [inline recipes/libs/compat/glibc.yaml:35]: AFAICT Ubuntu 18.04 is on 4.15.0...

**Comment by daxcore** [comment]
no c++17 issues! so i am fine with that solution too!

File: recipes/libs/compat/glibc.yaml (hunk 1 above)
Comment (mahaase) [reply recipes/libs/compat/glibc.yaml:35]: confirm.

**Comment by Ferruck** [comment]
I can confirm our AppImage building and working as before with these changes.

**Comment by jkloetzke** [comment]
Thanks!

**Pull request merged**